search/query.go

```diff
@@ -75,14 +71,15 @@ func doSearchProfiles(ctx context.Context, escli *es.Client, q string) (*EsSearc
 				"operator": "or",
 			},
 		},
+		"size": SearchDefaultSize,
```

Comment:
The removal of `escli.Search.WithSize(30)` surfaces back here for profile search.